scenes/quests/story_quests/stella/quest.tres

```diff
@@ -4,9 +4,11 @@
 
 [resource]
 script = ExtResource("1_sc4n7")
-title = ""
-description = ""
-authors = Array[String]([])
-affiliation = ""
+title = "Stella and the Missing Star"
+description = "Stella and the Missing Star follows a young girl named Stella as she journeys across the globe to restore a missing star from her favorite constellation. Guided by curiosity and gratitude, Stella travels to Guatemala, Northern Australia, and the Kalahari Desert, where she meets cultural guardians who each interpret the missing star through their own perspectives – a Mayan firekeeper, a Yolŋu fisherman, and a Khoisan nature guardian. Each encounter introduces a unique gameplay challenge – stealth, precision, and memory – that reflects the cultural story and deepens Stella’s understanding of the world. The game blends exploration, mythology, and personal growth to show how stories shape the way we see the stars – and ourselves.
```

Comment:
I think this is going to be way too long for the storybook. Let's see!

Reply:
I also fear that ŋ may not be in our font. It can also be spelled Yolngu https://en.wikipedia.org/wiki/Yolngu so we could change that if needed.

Reply:
It's not. Same story for the fancy dashes.